gazu/files.py

```diff
@@ -125,6 +125,16 @@ def get_all_working_files_for_entity(entity, task=None, name=None):
     return client.fetch_all(path, params)
 
 
+@cache
+def get_all_preview_files_for_task(task):
+    """
+    Retrieves all the preview files for a given task.
+    """
+    task = normalize_model_parameter(task)
+    path = "preview-files?task_id={}".format(task["id"])
```

Comment:
Instead of building the full path, you can use the `params` paramter of `fetch_all`.